### ISSUE TITLE
Default download folder

### ISSUE THREAD

**Issue by SynapseScribe** [ugetdm/uget#10]
There should be a setting to allow user to set a default download folder.
Even with "apply recent download settings", I get my download sometimes into C:\Users\<user>

**Comment by Futureknows** [comment]
Agreed this would be a very useful feature. It's surprising that it is missing, it's standard in virtually all downloading software.

**Comment by cgvirus** [comment]
It is there, But should be in settings tab as well
https://askubuntu.com/questions/489542/how-to-set-change-default-download-folder-for-uget
